spylon_kernel/init_spark_magic.py

```diff
@@ -31,9 +31,15 @@ def __init__(self, kernel):
         self.env['launcher'] = spylon.spark.launcher.SparkConfiguration()
         self.log = logging.Logger(self.__class__.__name__)
 
-    def cell_init_spark(self):
-        """Starts a SparkContext with a custom configuration defined
-        using Python code.
+    # Use argparse to parse the whitespace delimited cell magic options
+    # just as we would parse a command line.
+    @option(
+        "--stderr", action="store_true", default=False,
+        help="Capture stderr in the notebook instead of in the kernel log"
+    )
+    def cell_init_spark(self, stderr=False):
+        """%%init_spark --stderr CODE - starts a SparkContext with a custom
```

Comment:
CODE is the body of the cell containing the Python code to initialize the Spark context. I'll try to make it clearly in the description.